src/Streamly/Internal/Unicode/Stream.hs

```diff
@@ -1003,7 +1002,7 @@ encodeUtf8Lax = encodeUtf8
 -- "Haskell"
 --
 {-# INLINE fromStr# #-}
-fromStr# :: MonadIO m => Addr# -> SerialT m Char
+fromStr# :: MonadIO m => Addr# -> Stream.Stream m Char
```

Comment:
StreamD should be imported qualified and this type should be imported unqualified.